platform/src/common/components/Footer/.gitkeep

```diff
@@ -0,0 +1 @@
+Adds the empty folder to git history.
```

Comment:
Could you shed some light on these empty files? Anyway related to our file organisation?

Reply:
Hi @Baalmart 
.gitkeep files are a type of git files used to register empty directories to the VCS, just like the .gitignore files are for files to ignore.